### ISSUE TITLE
You might want to invest in margins/padding

### ISSUE THREAD

**Issue by That1M8Head** [0xCUB3/0xCUB3.github.io#1]
There's, like, no padding to the left or right of the website content, it doesn't look great. You may want to fix that.

**Comment by That1M8Head** [comment]
Here's what I mean:

![7824AD12-E585-4FAD-B8ED-6DCDF218C4F6](https://user-images.githubusercontent.com/16748384/165414227-546516eb-b45f-4e38-a235-08b73ad227ae.png)

**Comment by 0xCUB3** [comment]
yeah ik what you mean, I just haven't had the time to fix it yet. With AP exams and the end of the school year coming up, life's been hell lately

**Comment by 0xCUB3** [comment]
@That1M8Head sorry that I haven't updated it yet - I have finals and projects for the end of the academic year as well as other stuff going on in my life so I haven't had the time to fix the site yet. If you want, you can make a pull request with the fixes if I don't get around to fixing them soon

**Comment by That1M8Head** [comment]
Awesome, looks good 👍🏼
